packages/server/src/worker/workerTypes.ts

```diff
@@ -57,11 +58,26 @@ export interface TJobCheckLinksResult {
   kind: JobKind.CheckLinks;
 }
 
+/**
+ *  Update algolia index with latest resources
+ */
+export interface TJobSyncAlgoliaData {
+  kind: JobKind.SyncAlgolia;
+  userId: string; // _id of user who started this job
+}
+export interface TJobSyncAlgoliaResult {
+  kind: JobKind.SyncAlgolia;
+  jobName: string;
+  wasUpdated: boolean;
```

Comment:
would suggest a comment here in the type definition file for any params whose meaning isn't obvious.